onnxruntime/python/tools/transformers/optimizer.py

```diff
@@ -112,6 +112,10 @@ def optimize_by_onnxruntime(
         optimized_model_path = "{}_o{}_{}.onnx".format(path_prefix, opt_level, "gpu" if use_gpu else "cpu")
 
     sess_options.optimized_model_filepath = optimized_model_path
+    sess_options.add_session_config_entry(
+        "session.optimized_model_external_initializers_file_name", "external_initializers.bin"
```

Comment:
Could we use prefix of onnx model filename like "model.onnx.data" for "model.onnx"? It is because we do not want to overwrite the file when we save multiple optimized model in same directory.